git/Contributing.md

```diff
@@ -0,0 +1,58 @@
+# Contributing
+
+In this file is explained best practice for contributing on a Git version control repository hosting service such as _Github_, _Gitlab_, _Bitbucket_, etc.
+Also, there's explained flow for developing application.
+
+In flow given in post named [Successful Git Branching Model](http://nvie.com/posts/a-successful-git-branching-model/) there's detailed explanation of proposed flow, but we're trying to keep it short.
+
+## Branches
+
+One of the most powerful features of Git development flow are **branches**. After repository is created, `master` represents predefined branch. This branch is defined for _production_ version of application. Further more, it should be **protected** for force pushes (`git push origin master -f`).
```

Comment:
Use determiner `the` with words _repository_, _production_. Use `an` or `the` in front of word _application_.